org.envirocar.app/src/org/envirocar/app/views/recordingscreen/TrackMapFragment.java

```diff
@@ -343,7 +343,7 @@ public void onStyleLoaded(@NonNull Style style) {
                 }
             });
         } else {
-            Toast.makeText(getContext(), "Location access not granted", Toast.LENGTH_LONG).show();
+            showSnackbar("Location access not granted");
```

Comment:
Please, revert your changes but use getContext().getString(...) as text for Toast.makeText(...).